mars/dataframe/expressions/datasource/dataframe_from_tensor.py

```diff
@@ -0,0 +1,68 @@
+from ..core import DataFrameOperand, DataFrameOperandMixin
+from ..utils import parse_index
+from .... import opcodes as OperandDef
+from ....serialize import DataFrameField, SeriesField
+
+
+class TensorToDataFrame(DataFrameOperand, DataFrameOperandMixin):
+    """
+    Represents data from mars tensor
+    """
+    _op_type_ = OperandDef.DATAFRAME_FROM_TENSOR
+
+    _data = DataFrameField('data')
+    _dtypes = SeriesField('dtypes')
+
+    def __init__(self, data=None, dtypes=None, gpu=None, sparse=None, **kw):
+        if dtypes is None and data is not None:
+            dtypes = data.dtypes
+        super(TensorToDataFrame, self).__init__(_data=data, _dtypes=dtypes,
+                                                _gpu=gpu, _sparse=sparse, **kw)
+
+    @property
+    def data(self):
+        return self._data
+
+    @property
+    def dtypes(self):
+        return self._dtypes
+
+    def __call__(self, inputs, shape, index_value=None, columns_value=None, chunk_size=None):
+        if index_value is None and columns_value is None:
+            index_value = parse_index(self._data.index)
+            columns_value = parse_index(self._data.columns, store_data=True)
+
+        return self.new_dataframe(None, shape, dtypes=self.dtypes,
+                                  index_value=index_value,
+                                  columns_value=columns_value,
+                                  raw_chunk_size=chunk_size)
+
+    @classmethod
+    def tile(cls, op):
+        df = op.outputs[0]
+        in_df = op.data
```

Comment:
As said above, it's input is a tensor, the code should be `in_tensor = op.inputs[0]`.